src/parser.js

```diff
@@ -3,13 +3,25 @@
 const path = require("path");
 const Handlebars = require("handlebars");
 const logger = require("winston-color");
-import { defaultKeyValueMap, initOptions, checkMapTypes, convertValue } from "./options.js";
+const common = require("commonmark/lib/common.js");
```

Comment:
No it is not documented, nor exported by commonmark package.
I'll copy the implementation to our own code (and the license too - which appears to be 2-clause BSD).